lib/Minz/Session.php

```diff
@@ -4,18 +4,51 @@
  * La classe Session gère la session utilisateur
  */
 class Minz_Session {
+	private static $volatile = false;
+
+	/**
+	 * For mutual exclusion.
+	 */
+	private static $locked = false;
+
+	public static function lock() {
+		if (!self::$volatile && !self::$locked && session_start()) {
```

Comment:
It's the morning, but I don't understand why you need to add the `self::$locked` variable: doesn't `session_start` already acts as a lock?

Reply:
This is to prevent the case of calling several times `lock()` from the same process (already locked), but as we do not do it, it does not change anything at the moment